docs/extensibility.md

```diff
@@ -65,13 +65,13 @@ impl DiscoveryHandler for NessieDiscoveryHandler {
         if let Ok(_body) = hyper::Client::new().get(url).compat().await {
             // If the Nessie URL can be accessed, we will return a DiscoveryResult
             // instance
-            let props = HashMap::new();
+            let props: HashMap<String, String> = HashMap::new();

```

Comment:
does `use std::collections::HashMap;` need to be added as well?

Reply:
Yes, apolz.

Fixed: https://github.com/deislabs/akri/pull/71/commits/c78ded1fe6acaf94731dcf3aca851d976e4cfc61

Reply:
Added in my code although not reflected in this PR is the revised [discover](https://github.com/deislabs/akri/issues/72#issuecomment-716869659) function per #72.

I'm able to build `akri-agent` and `akri-controller` using @bfjelds  revised function